ui/modal_templates.go

```diff
@@ -596,6 +620,18 @@ func (m *ModalTemplate) handle(th *decredmaterial.Theme, load *modalLoad) (templ
 		}
 		template = m.securityToolsInfo()
 		return
+	case UnlockWalletRestoreTemplate:
+		if m.editorsNotEmpty(th, m.spendingPassword.Editor) && m.confirm.Button.Clicked() {
+			load.confirm.(func(string))(m.spendingPassword.Editor.Text())
+		}
+		if m.cancel.Button.Clicked() {
+			load.cancel.(func())()
```

Comment:
The unlock wallet modal doesn't seem to have a cancel button. There's no need to call a cancel function for it.